src/main/java/ua/bookstore/online/model/Role.java

```diff
@@ -0,0 +1,30 @@
+package ua.bookstore.online.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.security.core.GrantedAuthority;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "roles")
+public class Role implements GrantedAuthority {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Enumerated(EnumType.STRING)
+    private RoleName name;
```

Comment:
I suppose for better understanding of the code it is better to specify the constrains in the code not only in liquibase.
![image](https://github.com/k0sm0naft/java-online-book-store/assets/73792778/976c8e89-d15a-4918-a6d9-2f9950453841)

Reply:
You are right, I mees it.